src/Kotlin/KotlinLanguageService.kt

```diff
@@ -171,24 +164,46 @@ class KotlinLanguageService : LanguageService {
         }
     }
 
+    private fun ContentNode.renderAnnotationsForNode(node: DocumentationNode) {
+        node.annotations.forEach {
+            renderAnnotation(it)
+        }
+    }
+
+    private fun ContentNode.renderAnnotation(node: DocumentationNode) {
```

Comment:
We may try to put annotation(s) in square brackets for easier reading. There also could be cases when they are mandatory, not sure.